prow/crier/reporters/github/reporter.go

```diff
@@ -162,8 +165,61 @@ func (c *Client) Report(ctx context.Context, log *logrus.Entry, pj *v1.ProwJob)
 		defer lock.Release(1)
 	}
 
+	pjsToReport := func() []v1.ProwJob {
```

Comment:
This is a stylistic nit but I am neither a huge fan of inline functions nor of defining a function far away from its only callsite. There isn't really a need for this as the only inputn this needs is the prowjob or did I miss something?

Reply:
good idea, moved it out, will add unit test later